server/webapp/WEB-INF/rails.new/spec/controllers/api_v1/admin/pipelines_controller_spec.rb

```diff
@@ -365,6 +363,65 @@
       end
     end
 
+    describe :destroy do
```

Comment:
Please add the specs for security. It is already there for show, create and update in pipelines_controller_spec.rb